docs/docs-beta/docs/etl-pipeline-tutorial/refactor-your-project.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Refactor your project
+description: Refactor your completed project into a structure that is more organized and scalable. 
+last_update:
+  author: Alex Noonan
+sidebar_position: 80
+---
+
+Many engineers generally leave something alone once its working as expected. But the first time you do something is rarely the best implementation of a use case and all projects benefit from incremental improvements.
+
+## Splitting up project structure
+
+Right now the project is contained within one definitions file. This has gotten kinda unwieldy and if we were to add more to the project it would only get more disorganized. So we're going to create separate files for all the different Dagster core concepts: 
+
+- Assets
+- schedules
+- sensors
+- partitions
+
+The final project structure should look like this:
+```
+dagster-etl-tutorial/
+├── data/
+│   └── products.csv
+│   └── sales_data.csv
+│   └── sales_reps.csv
+│   └── sample_request/
+│       └── request.json
+├── etl_tutorial/
+│   └── assets.py
+│   └── definitions.py
+│   └── partitions.py
+│   └── schedules.py
+│   └── sensors.py
+├── pyproject.toml
+├── setup.cfg
+├── setup.py
+```
+
+### Assets
+
+Assets make up a majority of our project and this will be our largest file. 
+
+<CodeExample filePath="guides/tutorials/etl_tutorial_completed/etl_tutorial/assets.py" language="python" lineStart="1" lineEnd="292"/>
+
+### Schedules
+
+The schedules file will only contain the `weekly_update_schedule`.
+
+<CodeExample filePath="guides/tutorials/etl_tutorial_completed/etl_tutorial/schedules.py" language="python" lineStart="1" lineEnd="8"/>
+
+### Sensors
+
+The sensors file will have the job and sensor for the `adhoc_request` asset. 
+
+<CodeExample filePath="guides/tutorials/etl_tutorial_completed/etl_tutorial/sensors.py" language="python" lineStart="1" lineEnd="47"/>
+
+## Adjusting definitions object
+
+Now that we have separate files we need to adjust how the different elements are adding to definitions since they are in separate files 
+
+1. Imports
```

Comment:
I would make this section a note instead of a step:

> :::note
> The Dagster project runs from the root directory, so whenever you reference files in your project, you need to use the root as the starting point.
> Additionally, Dagster has functions to load all assets and asset checks from a module (`load_assets_from_modules` and `load_asset_checks_from_modules`, respectively).
> :::